httpserver/CMakeLists.txt

```diff
@@ -1,22 +1,22 @@
 cmake_minimum_required(VERSION 2.8.0)
+set(EXE loki-storage-server)
+project(${EXE})
 
-project(httpserver)
-
-add_executable(httpserver main.cpp http_connection.hpp)
-
+add_executable(${EXE} main.cpp http_connection.hpp)
+set(Boost_USE_STATIC_LIBS   ON)
```

Comment:
This is pretty undesirable for the debian packaged version: this makes the binaries much bigger, and make them use more memory (since shared boost object code can't be shared when statically linked).